aggsender/types/types.go

```diff
@@ -29,6 +29,8 @@ const (
 // AggsenderFlow is an interface that defines the methods to manage the flow of the AggSender
 // based on the different prover types
 type AggsenderFlow interface {
+	// CheckInitialStatus checks the initial status for the flow it's ok
+	CheckInitialStatus(ctx context.Context) error
```

Comment:
I would call it `CheckStartingBlock`:

```suggestion
	CheckStartingBlock(ctx context.Context) error
```

Reply:
I'll keep the name, because the idea is to check the initial Status of the flow, for aggchain is starting block, for PP is nothing... and who knowns if there are going to be another flow that check another thing... WDYT?